include/pnet_api.h

```diff
@@ -1067,12 +1069,14 @@ typedef struct pnet_ip_addr_t
 } pnet_cfg_ip_addr_t;
 
 /**
- * The Ethernet MAC address.
- */
-typedef struct pnet_ethaddr
-{
-  uint8_t addr[6];
-} pnet_ethaddr_t;
+ * Copy an IP address (as an integer) to a struct
+ *
+ * @param destination_struct  Out: destination
+ * @param ip                  In: IP address
+*/
+PNET_EXPORT void pnet_copy_ip_to_struct(
```

Comment:
Isn't this just a convenience function for the demo? Does it need to be in the stack? See above re avoiding osal.h.

Reply:
Moved to sampleapp